### ISSUE TITLE
How to install Dist::Zilla authordeps?

### ISSUE THREAD

**Issue by glasswalk3r** [jonasbn/github-action-perl-dist-zilla#21]
Hi there!

Thanks for developing this Github action!

I've being trying to use it with a repository of [mine](https://github.com/glasswalk3r/cpan-smoker-utils), but I found myself in trouble due `authordeps` from `Dist::Zilla`. Please ignore the missing `cpanfile` in the repository for now.

First I tried to generate a `cpanfile` as you have documented [here](http://jonasbn.github.io/til/dzil/cpanfile.html), by you!

Then, still running locally the Docker image, that's what I could achieve:

```
$ docker container run --rm -ti --entrypoint /bin/bash -v $PWD:/tmp jonasbn/ebirah:0.6.0
root@b8bf04da872c:/tmp# cpanm --installdeps .
--> Working on .
Configuring /tmp ... OK
==> Found dependencies: Test::Reporter::Transport::Socket, CPAN::Mini, YAML::XS, Test::Reporter, Term::ProgressBar, Set::Tiny, DBI
--> Working on Test::Reporter::Transport::Socket
Fetching http://www.cpan.org/authors/id/B/BI/BINGOS/Test-Reporter-Transport-Socket-1.02.tar.gz ... OK
Configuring Test-Reporter-Transport-Socket-1.02 ... OK
==> Found dependencies: Test::Reporter, Test::Reporter::Transport
--> Working on Test::Reporter
Fetching http://www.cpan.org/authors/id/D/DA/DAGOLDEN/Test-Reporter-1.62.tar.gz ... OK
Configuring Test-Reporter-1.62 ... OK
Building and testing Test-Reporter-1.62 ... OK
Successfully installed Test-Reporter-1.62
Building and testing Test-Reporter-Transport-Socket-1.02 ... OK
Successfully installed Test-Reporter-Transport-Socket-1.02
--> Working on CPAN::Mini
Fetching http://www.cpan.org/authors/id/R/RJ/RJBS/CPAN-Mini-1.111016.tar.gz ... OK
Configuring CPAN-Mini-1.111016 ... OK
Building and testing CPAN-Mini-1.111016 ... OK
Successfully installed CPAN-Mini-1.111016
--> Working on YAML::XS
Fetching http://www.cpan.org/authors/id/T/TI/TINITA/YAML-LibYAML-0.83.tar.gz ... OK
Configuring YAML-LibYAML-0.83 ... OK
Building and testing YAML-LibYAML-0.83 ... OK
Successfully installed YAML-LibYAML-0.83
--> Working on Term::ProgressBar
Fetching http://www.cpan.org/authors/id/M/MA/MANWAR/Term-ProgressBar-2.22.tar.gz ... OK
Configuring Term-ProgressBar-2.22 ... OK
==> Found dependencies: Class::MethodMaker
--> Working on Class::MethodMaker
Fetching http://www.cpan.org/authors/id/S/SC/SCHWIGON/class-methodmaker/Class-MethodMaker-2.24.tar.gz ... OK
Configuring Class-MethodMaker-2.24 ... OK
Building and testing Class-MethodMaker-2.24 ... OK
Successfully installed Class-MethodMaker-2.24
Building and testing Term-ProgressBar-2.22 ... OK
Successfully installed Term-ProgressBar-2.22
--> Working on Set::Tiny
Fetching http://www.cpan.org/authors/id/T/TR/TRENDELS/Set-Tiny-0.04.tar.gz ... OK
Configuring Set-Tiny-0.04 ... OK
Building and testing Set-Tiny-0.04 ... OK
Successfully installed Set-Tiny-0.04
--> Working on DBI
Fetching http://www.cpan.org/authors/id/T/TI/TIMB/DBI-1.643.tar.gz ... OK
Configuring DBI-1.643 ... OK
Building and testing DBI-1.643 ... OK
Successfully installed DBI-1.643
<== Installed dependencies for .. Finishing.
8 distributions installed
root@b8bf04da872c:/tmp# dzil test
Required plugin bundle Dist::Zilla::PluginBundle::Starter isn't installed.

Run 'dzil authordeps' to see a list of all required plugins.
You can pipe the list to your CPAN client to install or update them:

    dzil authordeps --missing | cpanm
```

So, my guess is that even though I can have a `cpanfile` versioned in the repository, I'll still be in trouble with the `authordeps`.

I'm willing to give you a hand with that, but not sure what is the best approach here. The tip from `dzil` usually does the trick nicely, but that would require the `entrypoint.sh` file to be changed.

**Comment by glasswalk3r** [comment]
See #22 for a proposal to fix that.

**Comment by jonasbn** [comment]
Hi @glasswalk3r 

I have merged #22 (by you) and have made a release 0.3.0, so this should be fixed now.

Thank you for your contribution, please let me know if you experience any issues.
